doc/whats_new/v0.24.rst

```diff
@@ -793,6 +799,14 @@ Changelog
   with different endianness.
   :pr:`17644` by :user:`Qi Zhang <qzhang90>`.
 
+- |Fix| :class:`sklearn.cross_decomposition.PLSSVD` accepts a `random_state`
+  parameter. This parameter is used in ``fit`` and ``fit_transform``
+  methods and initializes the `v0` parameter correctly with value sampled from
+  a uniform distribution in `[-1, 1]` to avoid convergence issues with another
+  initialization.
+  :pr:` #18302` by :user:`Gaurav Desai <gauravkdesai>` and
+  :user:`Ivan Panico <FollowKenny>`.
```

Comment:
Please remove this entry. It's partially duplicated from the one above and not related to the `sklearn.utils` module.

Reply:
@ogrisel note that I added an entry in `decomposition` and an entry in `Changes in model`.

Reply:
Ah right. Thanks.